src/main/java/jenkins/advancedqueue/JobGroup.java

```diff
@@ -140,7 +196,7 @@ public int getPriority() {
 	 * @deprecated Used in 2.x now replaced with dynamic {@link JobGroup#jobGroupStrategy}, will return the view
 	 */
 	@Deprecated
-        @CheckForNull
+    @CheckForNull
```

Comment:
Tabs vs. spaces in this and other files. Maybe it makes sense to just add a bulk refactoring in a follow-up pull request

Reply:
In the single commit https://github.com/jenkinsci/priority-sorter-plugin/pull/50/commits/89a142e35e8295cc4f2d10f9594f40bb615ae125
all tabs are replaced with whitespaces.